### PR TITLE
doc: copyediting, implement dev suggestions

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: doc/source/deployment/setup-deployer.rst [hunk 1]
```diff
@@ -68,11 +68,12 @@ Deployer. Set up your workspace with the following steps:
   export SOCOK8S_WORKSPACE_BASEDIR=~/socok8s-workspace
 
 
-Cloning repository
------------------------
+Cloning the repository
+----------------------
 
-To get started, clone the socok8s GitHub repository. This repository uses
```
Comment (jgu17) [inline doc/source/deployment/setup-deployer.rst]: @jsuchome should be able provide you information how to install socok8s thru Zypper install.

Comment (jsuchome) [reply doc/source/deployment/setup-deployer.rst]: Really, if this part is not for developers, tell users they can

1. install package from https://download.opensuse.org/repositories/Cloud:/socok8s/openSUSE_Leap_15
or 
2. from the iso image

File: doc/source/deployment/setup-deployer.rst [hunk 2]
```diff
@@ -176,7 +177,3 @@ You can improve SSH connections by enabling pipelining:
    [ssh_connection]
    pipelining = True
    EOF
-
-
-With this done, continue your deployment with the
-:ref:`configuredeployment` page.
```
Comment (jhesketh) [inline doc/source/deployment/setup-deployer.rst]: Why was this dropped?

Comment (csymons-suse) [reply doc/source/deployment/setup-deployer.rst]: It's the only page with such a statement. It's out of sequence with the procedure in the deployment guide. SES Configuration precedes Configure Cloud...Configure Cloud includes Configure for SES Integration, which states that the output of SES Integration must be present.

File: doc/source/deployment/setup-deployer.rst (hunk 1 above)
Comment (csymons-suse) [reply doc/source/deployment/setup-deployer.rst]: 1. available URLs at download.opensuse.org are openSUSE_Leap_15.0 or openSUSE_Leap_15.1. Should users be directed to 15.1?
In the src directory, there are RPMs for Helm, socok8s, and openSUSE-Addon-socok8s, as well as python libraries. Specifically what should a user install?
2. Where is the ISO image?

**Pull request merged**